serverless-logic/serverless-logic-supporting-services/serverless-logic-job-service.adoc

```diff
@@ -8,4 +8,24 @@ toc::[]
 
 The Job service schedules and executes tasks in a cloud environment. Independent services implement these tasks, which can be initiated through any of the supported interaction modes, including HTTP calls or Knative Events delivery.
 
-//I will include more modules in smaller, incremental PRs to avoid having one large PR. Currently building the directory structure for the OSL content. 
\ No newline at end of file
+In {ServerlessLogicProductName}, the Job service is responsible for controlling the execution of the time-triggered actions. Therefore all the time-based states that you can use in a workflow, are handled by the interaction between the workflow and the Job service.
+
+For example, every time the workflow execution reaches a state with a configured timeout, a corresponding job is created in the Job service, and when the timeout is met, a http callback is executed to notify the workflow.
+
+[NOTE]
+====
+If the project is not configured to use the Job Service, all time-based actions will use an in-memory implementation of that service. However, this setup must not be used in production, since every time the application is restarted, all the timers are lost, making it unsuitable for serverless architectures where applications might scale to zero at any time, etc.
+====
+
+The main goal of the Job service is to manage active jobs, such as scheduled jobs that need to be executed. Once a job reaches its final state, the Job service removes it. Ff you want to retain job information in a permanent repository, you can configure the Job service to produce status change events. This allows you to extend the jobs life span completion.
+
+[NOTE]
+====
+When you are using the {ServerlessOperatorName} to deploy workflows, there is no need to manually install or configure the Job service. The Operator handles these tasks automatically and manages all necessary configurations for each workflow to connect with it.
+====
+
+include::modules/serverless-logic-job-service-custom-execution.adoc[leveloffset=+1]
+include::modules/serverless-logic-job-service-common-config.adoc[leveloffset=+1]
```

Comment:
@kaldesai Similar comment for the serverless-logic-job-service-common-config.adoc document. Comments are good, but we don't make it available by now.